components/CostsTable.vue

```diff
@@ -127,7 +131,7 @@
         The assumed VSL for this country is {{ vslLabel(scenarios[0]) }}.
       </p>
       <p>
-        {{ scenarios.length > 1 ? 'These values' : 'This value' }}
+        {{ props.scenarios.length > 1 ? 'These values' : 'This value' }}
```

Comment:
Doesn't this switch need to use the same logic as that for "the current countries" vs "this country"? I'm currently looking at a comparison by disease but it says "These values.." under a list with the single country's vsl value.